bundle/Controller/Admin/FieldController.php

```diff
@@ -37,6 +37,7 @@ public function autoCompleteAction(Request $request): JsonResponse
         $searchResult = $this->tagsService->searchTags(
             $request->query->get('searchString') ?? '',
             $request->query->get('locale') ?? '',
+            showHidden: false,
```

Comment:
Should this be controlled through the configuration as well?

Reply:
True, but there might be a use case where you prepare the Tags and Content in advance, and just reveal the Tags at some time point. I'll leave it to @emodric.

cc @Ljudevit for info, maybe you have a use case for this?

Reply:
Yes, parameter is definitely needed.

Also, please do not use named parameters, at least not mixed like this, it just looks wrong :D Either change all of them to use named parameters, or none.